contents/docs/user-guides/groups.md

```diff
@@ -30,53 +43,51 @@ As another example, if `transaction` is a group type, then each `transactionID`
 
 ## Defining groups
 
-Groups are at the event level. If everything is setup correctly, then events that come into PostHog has a group associated with it, which is determined by you. See below for how to set this up. (link)
+Groups are at the event level. If everything is setup correctly, then events that come into PostHog have a group associated with them, which is determined by you. See below for how to set this up.
 
 Thus, there's no new events corresponding to groups: it's simply the events that already exist, aggregated by group values.
 
 A popular misconception is that groups are defined per user. This might make sense if the group type is a company, since a company is a set of users, but it quickly breaks down when you're looking at a transaction group type: here a user can do multiple transactions. Thus, remember that groups are defined on events.
```

Comment:
Note: It also breaks down if users belong to multiple companies or switch their active company. E.g. internal staff users or administrators for B2B usecases.

Reply:
Makes sense. I wanted to give an example that drives the point across, and transactions is more obvious. Not aiming to list everything wrong with groups per user here :)

Reply:
I'll just add another sentence to cover this